code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm

```diff
@@ -274,73 +116,89 @@
 	for (var/gas_id in moderator_internal.gases)
 		gas_power += (moderator_internal.gases[gas_id][GAS_META][META_GAS_FUSION_POWER] * moderator_internal.gases[gas_id][MOLES] * 0.75)
 
-	instability = MODULUS((gas_power * INSTABILITY_GAS_POWER_FACTOR)**2, toroidal_size) + (current_damper * 0.01) - iron_content * 0.05
+	instability = MODULUS((gas_power * INSTABILITY_GAS_POWER_FACTOR)**2, toroidal_size) + (current_damper * HYPERTORUS_INSTABILITY_DAMPER_FACTOR) - iron_content * HYPERTORUS_INSTABILITY_IRON_FACTOR
 	//Effective reaction instability (determines if the energy is used/released)
 	var/internal_instability = 0
-	if(instability * 0.5 < FUSION_INSTABILITY_ENDOTHERMALITY)
+	if(instability < FUSION_INSTABILITY_ENDOTHERMALITY)
 		internal_instability = 1
 	else
 		internal_instability = -1
 
 	/*
 	 *Modifiers
 	 */
-	///Those are the scaled gases that gets consumed and releases energy or help increase that energy
-	var/positive_modifiers = scaled_fuel_list[scaled_fuel_list[1]] + \
-								scaled_fuel_list[scaled_fuel_list[2]] + \
-								scaled_moderator_list[/datum/gas/nitrogen] * 0.35 + \
+	///Those are the scaled gases that gets consumed and adjust energy
+	// Gases that increase the amount of energy
+	var/energy_modifiers = scaled_moderator_list[/datum/gas/nitrogen] * 0.35 + \
 								scaled_moderator_list[/datum/gas/carbon_dioxide] * 0.55 + \
 								scaled_moderator_list[/datum/gas/nitrous_oxide] * 0.95 + \
 								scaled_moderator_list[/datum/gas/zauker] * 1.55 + \
-								scaled_moderator_list[/datum/gas/antinoblium] * 10 - \
-								scaled_moderator_list[/datum/gas/hypernoblium] * 10 //Hypernob decreases the amount of energy
-	///Those are the scaled gases that gets produced and consumes energy or help decrease that energy
-	var/negative_modifiers = scaled_fuel_list[scaled_fuel_list[3]] + \
+								scaled_moderator_list[/datum/gas/antinoblium] * 20
+	// Gases that decrease the amount of energy
+	energy_modifiers -= scaled_moderator_list[/datum/gas/hypernoblium] * 10 + \
 								scaled_moderator_list[/datum/gas/water_vapor] * 0.75 + \
 								scaled_moderator_list[/datum/gas/nitryl] * 0.15 + \
 								scaled_moderator_list[/datum/gas/healium] * 0.45 + \
```

Comment:
This isn't your fault, but god this code makes me sad

Reply:
aye, this is getting nuked from orbit in the content rework at least